src/languages/yaml.js

```diff
@@ -86,15 +86,21 @@ function(hljs) {
       },
       { // array listing
         className: 'bullet',
-        begin: '^ *-',
+      // TODO: remove |$ and |^ hacks when we have proper look-ahead support
+      begin: '((?<= )|^)?\\-(?=[ ]|$)',
```

Comment:
What is the first part of this regex doing again?

Reply:
It's a positive look-behind. It's saying the bullet must be preceded by a space, or be at the start of the line. That's my intent anyway, and the tests seemed to indicate it was working.

Reply:
I'll check this again actually - I'm not sure why I've made that look-behind optional...which seems like it defeats the purpose of it being there...